extensions/wikia/UserLogin/UserSignupSpecialController.class.php

```diff
@@ -122,82 +125,84 @@ public function signupForm() {
 		// process signup
 		$redirected = $this->request->getVal( 'redirected', '' );
 		if ( $this->wg->Request->wasPosted() && empty( $redirected ) ) {
+			$this->handleSignupFormSubmit();
+		} else {
+			$this->track( 'signup-start' );
```

Comment:
I guess my inclination is to make a method instance first and static if it has to be. Since this is only being called by instance methods, we know it could be instance, and it's not some utility method used outside of the class by other code. But I understand why it makes sense as static too. Just different preferences :)